### ISSUE TITLE
Inspect results of training

### ISSUE THREAD

**Issue by MislavSag** [a-hanf/mlr3automl#30]
Thanks for great package. I planned to write a package with AutoML for finace (investing) using mlr3 but it seems on first the you have already made great package (better than I woud be able to do for sure :)).

I have tried the package on my dataset with only one learner and Inf runtime. Here is simple code:
```
bmr_results = AutoML(my_task, learner_list = c("classif.ranger"), runtime = Inf)
bmr_results$train()
```
I don't understand how can inspect results  of the model after training?
I can see following methods and attributes:
```
names(bmr_results)
 [1] ".__enclos_env__"   "custom_trafo"      "additional_params" "portfolio"         "tuner"             "runtime"           "tuning_terminator"
 [8] "measure"           "resampling"        "preprocessing"     "learner"           "learner_timeout"   "learner_list"      "task"             
[15] "clone"             "initialize"        "tuned_params"      "resample"          "predict"           "train"        
```
I can't see aggregate method.
I have also tried to use resmaple method instead of train, but I got the same result.
Additionally I would like to if it is possible to use feature select steps in preprocessing?

**Comment by a-hanf** [comment]
Hi and thanks for your comment :)

The model is saved in the $learner slot. It is an `AutoTuner` from `mlr3tuning` ([link to docs](https://mlr3book.mlr-org.com/optimization.html#autotuner)) which wraps a `GraphLearner` from `mlr3pipelines` ([link to docs](https://mlr3book.mlr-org.com/pipelines.html)). You can use the $tuned_params method to view the hyperparameters selected during training:
`model$tuned_params()`. Let me know if there are any more questions around the methods and attributes.  

For feature selection there are multiple options. One is to create a feature selection pipeline using `mlr3pipelines` and `mlr3filters` like in this example: [link to docs](https://mlr3book.mlr-org.com/pipelines.html#feature-selection-pipeopfilter-and-pipeopselect). [mlr3fselect](https://github.com/mlr-org/mlr3fselect/) could also be interesting for you (not sure it would be as easy to integrate into your `mlr3automl` pipeline).

**Comment by MislavSag** [comment]
Thanks. In meantime I realized there is an archive attribute inside learner and also figured out there is an `tuned_params`.

I have already implemented filters from `mlr3filters` , but can't figure out how to implement `mlr3fselect` inside graph (preprocessing). I have tried to find example in `mlr3gallery` but I have only found examples without pipelines. In the end I have decided to to feature selection outside of pipes (graph) and than use important features inside AutoML.
